README.md

```diff
@@ -19,7 +20,7 @@
 
 ### Important Notes
 
-1. **YOU MUST READ [Contributing Code](https://github.com/go-gitea/gitea/wiki/Contributing-Code) BEFORE STARTING TO WORK ON A PULL REQUEST**.
+1. **YOU MUST READ [Contributing Code](https://github.com/go-gitea/gitea/CONTRIBUTING.md) BEFORE STARTING TO WORK ON A PULL REQUEST**.
```

Comment:
`You must read the [contributors guide]() before starting to work on a pull request`